Tests/Shared.Specs/CollectionEquivalencySpecs.cs

```diff
@@ -182,6 +183,34 @@ public void Add(Guid userId, params string[] roles)
             }
         }
 
+        [Fact]
+        public void When_BeEquivalentTo_calls_GetEqualityStrategy_it_should_be_thread_safe()
```

Comment:
🔧 I would not use literal names of code element in the test name. They tend to change over time. See also http://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html

Reply:
I agree.
I'll try to come up with something better.
Just chime in, if you have any suggestions or rules of thumb.